contrib/gitian-descriptors/gitian-linux-armhf.yml

```diff
@@ -0,0 +1,124 @@
+---
+name: "bitcoin-linux-armhf-0.13"
+enable_cache: true
+suites:
+- "trusty"
+architectures:
+- "amd64"
+packages:
```

Comment:
So curl is not needed here? 

https://github.com/bitcoin/bitcoin/blame/87d65622996d6b6539dba973a4427e542cc291c7/contrib/gitian-descriptors/gitian-linux.yml#L9